### PR TITLE
Add unassigner recipe

### PR DESCRIPTION
**Pull request by Ulthran**
Adds the [unassigner](https://github.com/PennChopMicrobiomeProgram/unassigner/tree/master) recipe.

This package is pure python but depends on vsearch, hopefully I specified that correctly in the recipe. Please let me know what to fix if not.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

### General instructions

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

### Instructions for avoiding API, ABI, and CLI breakage issues
Conda is able to record and lock (a.k.a. pin) dependency versions used at build time of other recipes.
This way, one can avoid that expectations of a downstream recipe with regards to API, ABI, or CLI are violated by later changes in the recipe.
If not already present in the meta.yaml, make sure to specify `run_exports` (see [here](https://bioconda.github.io/contributor/linting.html#missing-run-exports) for the rationale and comprehensive explanation).
Add a `run_exports` section like this:

```yaml
build:
  run_exports:
    - ...

```

with `...` being one of:

| Case                             | run_exports statement                                               |
| -------------------------------- | ------------------------------------------------------------------- |
| semantic versioning              | `{{ pin_subpackage("myrecipe", max_pin="x") }}`     |
| semantic versioning (0.x.x)      | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}`   |
| known breakage in minor versions | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| known breakage in patch versions | `{{ pin_subpackage("myrecipe", max_pin="x.x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| calendar versioning              | `{{ pin_subpackage("myrecipe", max_pin=None) }}`    |

while replacing `"myrecipe"` with either `name` if a `name|lower` variable is defined in your recipe or with the lowercase name of the package in quotes.

### Bot commands for PR management

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/unassigner/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+  run_exports:
+    - {{ pin_subpackage('unassigner', max_pin="x") }}
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
```
Comment (thanhleviet) [inline recipes/unassigner/meta.yaml:26]:
```suggestion
    - python
    - vsearch
    - biopython
    - scipy

```

File: recipes/unassigner/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 0a79a0fdf88318a2583e2a46deffb6f6ff4ce612c55b470c007eed7bb9aa2194
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - unassigner = unassigner.command:main
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (thanhleviet) [inline recipes/unassigner/meta.yaml:17]:
```suggestion
  script: {{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv
```

File: recipes/unassigner/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+test:
+  imports:
+    - unassigner
+  commands:
+    - unassigner --help
+
+about:
+  home: https://github.com/PennChopMicrobiomeProgram/unassigner
+  license: MIT
+  license_family: MIT
```
Comment (thanhleviet) [inline recipes/unassigner/meta.yaml:38]: It's `GPLv2` in https://github.com/PennChopMicrobiomeProgram/unassigner/blob/23ddca1c270d8a31067229d0f912aafbed8e6d75/setup.py#L39
is it possible to add a LICENSE file to the original repo as well ?

**Comment by Ulthran** [comment]
Thanks @thanhleviet just merged the suggested changes.

**Review by thanhleviet — APPROVED** [review]

File: recipes/unassigner/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  commands:
+    - unassigner --help
+
+about:
+  home: https://github.com/PennChopMicrobiomeProgram/unassigner
+  license: GPL-2.0-or-later
+  license_family: GPL
+  license_file: 
+  summary: "Type strain identification for 16S reads"
+  doc_url:
```
Comment (bgruening) [inline recipes/unassigner/meta.yaml:43]: Do you want to add here something? If not please clean it up.

**Pull request merged**